src/components/Form/Form.scss

```diff
@@ -0,0 +1,42 @@
+.register-form {
+  height: calc(100% - 118px);
+  width: 100%;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+
+  &__header {
+    margin-bottom: 30px;
+  }
+
+  &__inputs {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  &__progress {
+    position: relative;
+  }
+
+  &__button {
+    font-size: 25px;
+    color: white;
+    background: linear-gradient(to right, $darkAccent 20%, $lightAccent 300%);
+    width: 100%;
+    height: 80px;
+  }
+}
+
+.test {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 5px;
+  background: $lightAccent;
+  transform-origin: left;
+  animation-duration: 0.3s;
+  animation-timing-function: ease-out;
+}
```

Comment:
This is some SCSS magic, great work!

Reply:
What the hell is going on here..Looks great!!!

Reply:
Looks great!